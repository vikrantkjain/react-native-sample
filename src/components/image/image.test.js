import React from 'react';
import renderer from 'react-test-renderer';
import Image from './index';
// import {Images} from '../../constants';

const url =
  'https://cdn.kinandcarta.com/-/media-assets/images/kincarta/insights/2022/02/react-native/react_hero.png?as=0&iar=0&w=1200&rev=61e1dad3af7e465e9544cf8490237772&hash=0AD31383BCBA1DA1C88546327312BA33';

jest.mock('react-native-config', () => 'RNConfig');
test('renders correctly', () => {
  const tree = renderer.create(<Image source={{uri: url}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
