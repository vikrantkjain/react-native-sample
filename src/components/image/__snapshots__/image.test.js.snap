// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "overflow": "hidden",
      },
      undefined,
    ]
  }
>
  <FastImageView
    defaultSource={null}
    resizeMode="cover"
    source={
      {
        "uri": "https://cdn.kinandcarta.com/-/media-assets/images/kincarta/insights/2022/02/react-native/react_hero.png?as=0&iar=0&w=1200&rev=61e1dad3af7e465e9544cf8490237772&hash=0AD31383BCBA1DA1C88546327312BA33",
      }
    }
    style={
      {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</View>
`;
